Sources/NostrSDK/Events/Tags/PubkeyTag.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  PubkeyTag.swift
+//  
+//
+//  Created by Terry Yiu on 11/15/23.
+//
+
+import Foundation
+
+/// A public key tag.
+public protocol PubkeyTag {
```

Comment:
Since this protocol only requires that conformers provide a pubkey and not that they are a `Tag`, maybe this could be `PubkeyProviding`?